ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -44,7 +44,7 @@
     "DisplayContext": "MANDATORY",
     "PageID": "defenceAdmittedPartRoute",
     "ShowSummaryChangeOption": "Y",
-    "FieldShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND specDefenceAdmittedRequired = \"Yes\"",
+    "FieldShowCondition": "specDefenceAdmittedRequired = \"Yes\"",
```

Comment:
This condition is in the same backtracking problem case than the one we talk yesterday. Even if specDefenceAdmittedRequired can only be filled in when you select part admission, you can then go back, choose another admit/reject option, and when you go forward this screen will appear

Reply:
i test this and it seems to work ok. When I did the PART_ADMISSION I could see it on Check Answer Page, when go back and check PART_ADMISSION to for example FULL_DEFENCE I did not see this answer on the Check Answer Page, so I think that confirm, it work like it should, am I right?